packages/shared/package.json

```diff
@@ -32,6 +32,7 @@
     "test:ci": "pnpm test:only"
   },
   "devDependencies": {
+    "@logto/connector-kit": "workspace:*",
```

Comment:
Is this necessary?

Reply:
We can not mock @logto/connector-kit methods with out this dependency. So seems it's necessary.